ui/components/component-library/base-icon/README.mdx

```diff
@@ -0,0 +1,79 @@
+import { Story, Canvas, ArgsTable } from '@storybook/addon-docs';
+
+import { BaseIcon } from './base-icon';
+
+### This is a base component. It should not be used in your feature code directly but as a "base" for other UI components
+
+# BaseIcon
+
+The `BaseIcon` is the base component for all icons. It is used in conjunction with a script to create all icons it should not be used directly.
+
+<Canvas>
+  <Story id="ui-components-component-library-base-icon-base-icon-stories-js--default-story" />
+</Canvas>
+
+## Props
+
+The `BaseIcon` accepts all props below as well as all [Box](/ui-components-ui-box-box-stories-js--default-story) component props
+
+<ArgsTable of={BaseIcon} />
+
+### Size
+
+Use the `size` prop to set the size of the `BaseIcon`
```

Comment:
Mention the `SIZES` object here

Reply:
Updated https://github.com/MetaMask/metamask-extension/pull/15609/commits/8b173b3ba673d1a4ef2b82e93dc4f5eb8fd4a93c